src/components/screens/screens-screen.tsx

```diff
@@ -1,12 +1,10 @@
-import strings from "../../localization/strings"
+import strings from "../../localization/strings";
 
 /**
  * Renders screens screen
  */
 const ScreensScreen = () => {
-  return (
-    <div>{strings.generic.notImplemented}</div>
-  )
+  return <div>{strings.generic.notImplemented}</div>;
```

Comment:
Should this be renamed to `devices-screen.tsx` to maintain consistency of terms across the whole project?

Reply:
Fixed